manifests/apache_exporter.pp

```diff
@@ -133,5 +135,8 @@
     service_ensure     => $service_ensure,
     service_enable     => $service_enable,
     manage_service     => $manage_service,
+    export_scrape_job  => false,
```

Comment:
Should this be `export_scrape_job => $export_scrape_job` with `Boolean $export_scrape_job = false,` added to the class parameters?

Reply:
oh, jeez, yes! that's embarassing... :see_no_evil: